rpi/boundary/car_event_processor.py

```diff
@@ -23,29 +23,40 @@ def __init__(self):
 
         dispatcher.connect(self.move_forward, signal=CarStatus.MOVE_FORWARD, sender=dispatcher.Any)
 
+        self._socketio = socketio
+
     def take_picture(self):
```

Comment:
Try to use **emit** instead of **send**.
For me **send** doesn't work.

Reply:
Unfortunately, it doesn't make any difference. I also did try to set the broadcast parameter to true or to emit on a specific event and use that event on client side. It simply just appear sometimes in browser. I did try to look in app console, after activating the logs, and browser console to see if I see anything special occurs but I did not see